client/components/jetpack/card/jetpack-product-card/display-price/paid.tsx

```diff
@@ -98,8 +101,8 @@ const Paid: React.FC< OwnProps > = ( props ) => {
 		tooltipText,
 		displayPriceText,
 	} = props;
-	const finalPrice = ( discountedPrice ?? originalPrice ) as number;
-	const isDiscounted = !! ( finalPrice && originalPrice && finalPrice < originalPrice );
+	const finalPrice = ( isNumber( discountedPrice ) ? discountedPrice : originalPrice ) as number;
```

Comment:
`In a lot of places the discounted price was checked by a boolean check, which I changed to an isNumber check because 0 would be falsy`

I suppose this is one such place. When you have it as an existing ?? check, what was the behaviour?

Reply:
>  When you have it as an existing ?? check, what was the behaviour?

Well, then with the Advanced plan where `discountedPrice is 0`, it the `discountedPrice ?? originalPrice` would be `0 ?? 4799` which would be `4799` which is wrong. 0 is falsy in JS that's why we need the isNumber check